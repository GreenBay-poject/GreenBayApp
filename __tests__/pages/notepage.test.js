import React from 'react';
import renderer from 'react-test-renderer';
import NotePage from '../../pages/note_page'

describe('<NotePage />', () => {
    it('Sample NotePage', () => {
        const tree = renderer.create(<NotePage />).toJSON();
        expect(tree).toMatchSnapshot();
        //expect(tree.children[0].children[0].type).toBe('View');
        //expect(tree.children[0].children[1].type).toBe('View');
        //console.log(tree)

    });
});