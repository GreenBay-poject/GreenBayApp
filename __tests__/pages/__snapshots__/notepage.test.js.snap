// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotePage /> Sample NotePage 1`] = `
<RCTScrollView>
  <View>
    <View />
    <ActivityIndicator
      animating={true}
      color="#C00"
      hidesWhenStopped={true}
      size="large"
      style={
        Array [
          Object {
            "height": 80,
          },
        ]
      }
    />
  </View>
</RCTScrollView>
`;
